README.rst

```diff
@@ -10,11 +10,11 @@ Why Should I Use This?
 ----------------------
 
 The biggest reason to use twine is that it securely authenticates you to PyPI
-over HTTPS using a verified connection while ``python setup.py upload`` `uses
-HTTP and exposes your credentials <http://bugs.python.org/issue12226>`_.  This
-means anytime you use it you expose your username and password to being
-sniffed. Twine uses only verified TLS to upload to PyPI protecting your
-credentials from theft.
+over HTTPS using a verified connection while ``python setup.py upload`` `only
+recently stopped using HTTP <http://bugs.python.org/issue12226>`_.  This means
```

Comment:
"Only recently" is deceptive and either needs to be expanded or there needs to be a footnote explaining what that even means. Expecting people to read a bpo issue to try to determine when it stopped does not seem reasonable to me. Further, not using HTTP simply isn't enough to make distutils secure.